pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -787,32 +788,38 @@
   cs_ambassador_steps_heading: "What do I need to become a CS Ambassador?"
   cs_ambassador_steps_list_title_01: "Passion"
   cs_ambassador_steps_list_title_02: "Experience in a CS Course"
   cs_ambassador_steps_list_title_03: "School that offers CS"
   cs_ambassador_steps_list_desc_01: "A passion for encouraging young women at your school to take computer science classes"
-  cs_ambassador_steps_list_desc_02: "Have completed or is currently enrolled in a year-long computer science class (recommended: CS Principles or AP® CSA)"
-  cs_ambassador_steps_list_desc_03: "Attend a high school that regularly offers a year-long computer science course (recommended: CS Principles or AP® CSA)"
+  cs_ambassador_steps_list_desc_02: "Have taken a year-long computer science class (or be currently enrolled in one)."
+  cs_ambassador_steps_list_desc_03: "Attend a high school that regularly offers a year-long computer science course."
   cs_ambassador_benefits_heading: "Program benefits"
   cs_ambassador_benefits_list_title_01: "National community"
-  cs_ambassador_benefits_list_title_02: "Community service"
-  cs_ambassador_benefits_list_title_03: "Recognition"
-  cs_ambassador_benefits_list_title_04: "Swag"
-  cs_ambassador_benefits_list_title_05: "Certificate"
-  cs_ambassador_benefits_list_title_06: "Experiences"
-  cs_ambassador_benefits_list_desc_01: "Access to a national community of young leaders committed to closing the gender gap in computer science."
-  cs_ambassador_benefits_list_desc_02: "Community service hours and the opportunity to earn the [U.S. Presidential Volunteer Service Award](%{award_url})."
-  cs_ambassador_benefits_list_desc_03: "Recognition as a 2023-2024 Ambassador and eligible to be honored in the Code.org Ambassador Hall of Fame."
-  cs_ambassador_benefits_list_desc_04: "Be identified as a CS Ambassador through special Code.org branded apparel, stickers, and more!"
-  cs_ambassador_benefits_list_desc_05: "Official certificate of participation from Code.org."
-  cs_ambassador_benefits_list_desc_06: "Gain experiences and stories to share on your college application, resume and more!"
+  cs_ambassador_benefits_list_title_02: "Recognition"
+  cs_ambassador_benefits_list_title_03: "Resources"
+  cs_ambassador_benefits_list_title_04: "Opportunities"
+  cs_ambassador_benefits_list_desc_01: "Join a national network of young leaders committed to closing the gender gap in computer science."
+  cs_ambassador_benefits_list_desc_02: "Official recognition as a Code.org CS Ambassador to share on your college application, resume and more!"
```

Comment:
Nit: would you mind adding a comma after "resume" please

Reply:
Because I was just following the spec this feels like a design question (not sure how everyone feels about the oxford comma😂)- @dmcavoy is there any wild chance we have a standard for this?

Reply:
Added the oxford comma! I prefer it as well:)